scripts/identify-web-features.js

```diff
@@ -0,0 +1,303 @@
+function getMarkdownContentForFeature(feature) {
+  let str = `### Feature **${escapeFeatureName(feature)}**\n\n`;
+  str += `* **ID:** ${feature.id}\n`;
+  str += `* **Name:** ${escapeFeatureName(feature)}\n`;
+  str += `* **Description:** ${feature.description_html}\n`;
+  str += `* **Baseline status:** ${printBaselineStatus(feature)}\n`;
+  str += printDocs(feature);
+  str += printStandardPositions(feature);
+  str += printUseCounter(feature);
+  str += printSurveys(feature);
+  str += printPreviousInterops(feature);
+  str += printWPTLink(feature);
+  str += `* **More information:** See the [web-features explorer](https://web-platform-dx.github.io/web-features-explorer/features/${feature.id}/).\n\n`;
+
+  return str;
+}
+
+// Post a new comment with the given markdown content or update an existing comment if it already exists.
+async function postOrUpdateComment(issueNumber, markdown) {
+  // Retrieve existing comments to check if we already posted a comment.
+  const commentsResponse = await octokit.request(`GET /repos/${argv.repo}/issues/${issueNumber}/comments`, {
+    headers: {
+      "X-GitHub-Api-Version": GITHUB_API_VERSION
+    }
+  });
+  const existingComment = commentsResponse.data.find(comment => comment.body.includes(HIDDEN_COMMENT_IN_ISSUE));
+
+  if (existingComment) {
+    // The bot already posted a comment. Update it.
+    console.log(`Updating existing comment #${existingComment.id}...`);
+    await octokit.request(`PATCH /repos/${argv.repo}/issues/comments/${existingComment.id}`, {
+      body: markdown,
+      headers: {
+        "X-GitHub-Api-Version": GITHUB_API_VERSION
+      }
+    });
+  } else {
+    // Post a new comment.
+    console.log(`Posting a new comment...`);
+    await octokit.request(`POST /repos/${argv.repo}/issues/${issueNumber}/comments`, {
+      body: markdown,
+      headers: {
+        "X-GitHub-Api-Version": GITHUB_API_VERSION
+      }
+    });
+  }
+}
+
+// The main entry point to the script.
+async function main() {
+  const issue = await getReferencedIssue();
+
+  console.log(`Processing issue #${issue.number}: "${issue.title}"`);
+  const featureIds = findFeaturesInIssue(issue);
+  const features = await Promise.all(featureIds.map(id => getFeatureData(id)));
+
+  if (features.length === 0) {
+    console.log("Could not find any matching features the issue body.");
+    return;
+  }
+
+  console.log(`Found ${features.length} matching feature(s):`);
+  console.log(features.map(f => `- ${f.id}`).join("\n"));
+
+  let content = "_This comment was automatically generated based on the information you provided. Please don't edit it._\n\n";
+  content += `Below is additional information about the web feature${features.length > 1 ? "s" : ""} (from the [web-features project](https://github.com/web-platform-dx/web-features/)) which ${features.length > 1 ? "are" : "is"} referenced in your proposal.`;
+  content += `This information might help motivate your focus area proposal.\n\n`;
```

Comment:
I don't think we need this line; I think the point of this is to tell _us_ which features are being proposed. I think we need something informing the user what to do if the output is _wrong_, especially if no features are found (which we should also handle below).

Something like:
"If these web features do not accurately correspond to your proposal, please add a comment below, or update your initial comment to include the string `web-features: <feature name>`"

(that assumes that we'll have some explicit web-feature id that overrides any other fields).